src/routes/app-routes.tsx

```diff
@@ -162,6 +113,17 @@ export function AppRoutes(): JSX.Element {
           </AccountGate>
         }
       />
+      <Route
+        path={RouteUrls.ViewSecretKey}
+        element={
+          <AccountGate>
+            <SaveSecretKey />
+          </AccountGate>
+        }
+      />
+      <Route path={RouteUrls.Unlock} element={<Unlock />} />
+      {/* Catch-all route redirects to onboarding */}
+      <Route path="*" element={<Onboarding />} />
```

Comment:
Shouldn't this be a navigation event to `RouteUrls.Onboarding`, as this is a second route declaration of the same page?

Reply:
O yeah, good point. I will change it. Thanks.